### PR TITLE
2.x Fix for ticket #266

### PR DESCRIPTION
**Pull request by fa-dd**
modified file services.xml to remove strict requirement

### CODE REVIEW

File: Tests/Locale/DefaultProviderTest.php [hunk 1]
```diff
@@ -43,7 +43,7 @@ public function testDefaultLocaleIsInLocales()
 
         // set expectations for constructor calls
         $this->setExpectedException(
-            'InvalidArgumentException', 'Default locale should be contained in locales'
+            'InvalidArgumentException', 'Default locale `de` should be contained in locales'
```
Comment (jordisala1991) [inline Tests/Locale/DefaultProviderTest.php]: Why are you changing this line?

Comment (fa-dd) [reply Tests/Locale/DefaultProviderTest.php]: Normally it should't be in the PR. it was only to check if the correct test was used, as I had a doubt about it.
Only change needed for the fix was in services.xml

Comment (jordisala1991) [reply Tests/Locale/DefaultProviderTest.php]: Revert this change then

**Comment by fa-dd** [comment]
Will close this one, and do a new one with the correct commit (have to do a new fork to get a correct base again)

**Pull request closed without merging**